CMakeLists.txt

```diff
@@ -184,13 +193,30 @@ function(build_GenomicsDB_executable target)
     if(LIBCSV_FOUND)
         target_link_libraries(${target} ${LIBCSV_LIBRARY})
     endif()
+    if(LIBDBI_FOUND)
+        target_link_libraries(${target} ${LIBDBI_DEV_LIBRARY} ${LIBPGSQL_DRIVER_LIBRARY})
+    endif()
     target_link_libraries(${target} ${OPENSSL_LIBRARIES} ${ZLIB_LIBRARIES} ${PROTOBUF_LIBRARIES})
     if(LIBRT_LIBRARY)
         target_link_libraries(${target} ${LIBRT_LIBRARY})
     endif()
     install(TARGETS ${target} RUNTIME DESTINATION bin)
 endfunction()
 
+function(build_GenomicsDB_executable target)
+    add_executable(${target} src/${target}.cc)
+    build_GenomicsDB_executable_common(${target})
+endfunction()
+
+function(build_GenomicsDB_test_executable target src_file_list)
+    find_package(GTest)
+    if(GTEST_FOUND)
+        add_executable(${target} ${src_file_list})
+        target_link_libraries(${target} ${GTEST_BOTH_LIBRARIES})
+        build_GenomicsDB_executable_common(${target})
+    endif()
+endfunction()
+
```

Comment:
Added gtest installation in .travis.yml
LD_LIBRARY_PATH and CLASSPATH already exist
install TARGET is on line 203